app/authorize.ejs

```diff
@@ -13,11 +13,9 @@
 function removeFromStorage(key) {
     return localStorage.removeItem(key);
 }
-var trustedHosts = [
-    [/^staging\.cbd\.int$/i, /.*\.staging\.cbd\.int$/i],
-    [/^cbddev\.xyz$/i, /.*\.cbddev\.xyz$/i],
-    [/^cbd\.int$/i, /.*\.cbd\.int$/i]
-];
+
+var trustedHosts = [<%=trustedDomains%>];
```

Comment:
I am not sure what difference will it make?

Reply:
it will not be possible to access `cbd.int` token from `cbddev.xyz` domain & vice-versa